src/libs/LoginUtils.ts

```diff
@@ -75,4 +76,19 @@ function areEmailsFromSamePrivateDomain(email1: string, email2: string): boolean
     return Str.extractEmailDomain(email1).toLowerCase() === Str.extractEmailDomain(email2).toLowerCase();
 }
 
-export {getPhoneNumberWithoutSpecialChars, appendCountryCode, isEmailPublicDomain, validateNumber, getPhoneLogin, areEmailsFromSamePrivateDomain};
+function fetchSAMLUrl(body: FormData): Promise<Response | void> {
+    return fetch(CONFIG.EXPENSIFY.SAML_URL, {
+        method: CONST.NETWORK.METHOD.POST,
+        body,
+        credentials: 'omit',
+    })
+        .then((response) => {
+            if (!response.ok) {
+                throw new Error('An error occurred while logging in. Please try again');
+            }
+            return response.json() as Promise<Response>;
+        })
+        .then((response) => response);
```

Comment:
Yeah, at least from my testing we need both - I also followed the convention from [here](https://github.com/Expensify/App/blob/7be9987ce91638600b5f61c8652c50517c2ca001/src/libs/HttpUtils.ts#L54)

Reply:
What's the convention in this case? I don't see `.then((response) => response);` in the linked code, and was also wondering about this because this line looks like no-op

Reply:
looks like this is actually fine without - removing!